.github/workflows/build.yml

```diff
@@ -47,13 +47,16 @@ jobs:
       - id: setup-python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
+          python-version: "3.11"
```

Comment:
I believe this should be done to the `test` job as well:

https://github.com/cisagov/skeleton-docker/blob/201692f8e35feff3badc715fc74d7bef1e3ba308/.github/workflows/build.yml#L306-L309

Reply:
_I'll do it myself_ [18c3bc4](https://github.com/cisagov/skeleton-docker/pull/164/commits/18c3bc4cd9251631448e089bd7baa3bccc6694dc).